ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/131__rhods_operatorv2.robot

```diff
@@ -0,0 +1,180 @@
+    Component Should Not Be Enabled    dashboard
+    Component Should Not Be Enabled    distributedWorkloads
+    Component Should Not Be Enabled    kserve
+    Component Should Not Be Enabled    modelmeshserving
+    Component Should Not Be Enabled    workbenches
+
+
+*** Keywords ***
+Verify Component Resources
+    [Documentation]    Currently always fails, need a better way to check
+    [Arguments]    ${component}
+    Enable Single Component    ${component}
+    ${filepath} =    Set Variable    ods_ci/tests/Resources/Files/operatorV2/
+    ${expected} =    Get File    ${filepath}${component}.txt
+    Run    oc get $(oc api-resources --namespaced=true --verbs=list -o name | awk '{printf "%s%s",sep,$0;sep=","}') --ignore-not-found -n redhat-ods-applications -o=custom-columns=KIND:.kind,NAME:.metadata.name | sort -k1,1 -k2,2 | grep -v "PackageManifest\\|Event\\|ClusterServiceVersion" > ${filepath}${component}_runtime.txt  # robocop: disable
+    Process Resource List    filename_in=${filepath}${component}_runtime.txt
+    ...    filename_out=${filepath}${component}_processed.txt
+    ${actual} =    Get File    ${filepath}${component}_processed.txt
+    Remove File    ${filepath}${component}_processed.txt
+    Remove File    ${filepath}${component}_runtime.txt
+    Should Be Equal As Strings    ${expected}    ${actual}
+
+Enable Single Component
+    [Documentation]    Enables a single component AND disables all other components. If "none" is used
+    ...    disables all components
+    [Arguments]    ${component}    ${dsc_name}=default
+    IF    "${component}" not in @{COMPONENTS} and "${component}" != "none"
+        Log    unknown component: ${component}
+        RETURN
+    END
+    ${len} =    Get Length    ${COMPONENTS}
+    ${patch} =    Set Variable    ${PATCH_PREFIX}
+    FOR    ${index}    ${cmp}    IN ENUMERATE    @{COMPONENTS}
+        IF     "${cmp}"=="${component}"
+            ${sub} =    Change Component Status    component=${cmp}    enable=true    run=${FALSE}
+        ELSE
+            ${sub} =    Change Component Status    component=${cmp}    enable=false    run=${FALSE}
+        END
+        IF    ${index} ==0
+            ${patch} =    Catenate    SEPARATOR=    ${patch}    ${sub}
+        ELSE
+            ${patch} =    Catenate    SEPARATOR=,${SPACE}    ${patch}    ${sub}
+        END
+    END
+    ${patch} =    Catenate    SEPARATOR=    ${patch}    }}}'
+    Log    ${patch}
+    ${status} =    Run    ${patch}
+    Log    ${status}
+    Sleep    30s
+
+Change Component Status
+    [Documentation]    Enables or disables a single component. Can either run the patch command directly (if `run` is
+    ...    set to true) or return the patch string to be combined later for a bigger patch command.
+    [Arguments]    ${component}    ${run}=${TRUE}    ${enable}=true
+    IF    "${component}" not in @{COMPONENTS}
+        Log    unknown component: ${component}
+        RETURN
```

Comment:
should we fail in this case?

Reply:
right right. Maybe, could you set the log level to WARN? so we may see soon if there's something wrong with the test itself in the future

Reply:
updated to log level warn